third_party/python.bzl

```diff
@@ -44,6 +44,16 @@ def tensorboard_python_workspace():
         build_file = str(Label("//third_party:urllib3.BUILD")),
     )
 
+    http_archive(
+        name = "org_pythonhosted_attrs",
```

Comment:
[AI REQUIRED] Is this going to be the required dependency on TB? If so, add a line here
https://github.com/tensorflow/tensorboard/blob/6359c4492e8844ace5761c4ec661dd1e5b469919/tensorboard/pip_package/setup.py#L25-L38